src/election_anomaly/user_interface/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from election_anomaly import db_routines
 from election_anomaly import db_routines as dbr
-from election_anomaly.db_routines import create_cdf_db as db_cdf
+from election_anomaly.db_routines import Create_CDF_db as db_cdf
```

Comment:
I don't believe this is correct. The latest version of master should only ever reference `create_cdf_db`, not `Create_CDF_db`. If you are using the `python setup.py install` approach, you may need to rebuild the package and try again (in fact, you'll need to rebuild each time you make _any_ change).